server.js

```diff
@@ -607,6 +607,9 @@ app.use('/pl/course_instance/:course_instance_id/clientFilesCourse', [
     require('./middlewares/studentAssessmentAccess'),
     require('./pages/clientFilesCourse/clientFilesCourse'),
 ]);
+app.use('/pl/course/:course_id/clientFilesCourse', [
+    require('./pages/clientFilesCourse/clientFilesCourse'),
+]);
```

Comment:
I think this route should be needed. For all student pages you will have a valid `locals.course_instance.id` so you can use the route `/pl/course_instance/:course_instance_id/clientFilesCourse`. Or am I misunderstanding why you needed to add this route?

Reply:
I think this course is needed because it is possible to view a course's questions without a :course_instance_id, in which case this route is needed. For example, clicking on 'exampleCourse' and then 'questions' in the top navbar brings you to the questions page without a course_instance_id